CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Changed
+- Update links in `README.md` to PULP-platform
```

Comment:
```suggestion
```

I suggest to only track changes that directly affect users of a program or library in the changelog. Changes to tests, CI, documentation, and so on should not be noted in the changelog IMO, because they distract users from changes that directly affect them.